docs/Migration-guides.md

```diff
@@ -0,0 +1,100 @@
+# Migration guides
+
+## Migrating from `1.0.0-rc.4` to `1.0.0-rc.5`
+
+### New npm package: `@ui5/webcomponents-icons`
+
+All individual icons were moved to a new npm package called `@ui5/webcomponents-icons`.
+
+If you are using icons in your project:
+
+1. Add the new package as a dependency to your project
+
+	`npm i @ui5/webcomponents-icons`
+	
+	or
+	
+	`yarn add @ui5/webcomponents-icons`
+	
+	depending on your package manager.
+	
+2. Change the imports for the individual icons (if you were importing them one by one):
+
+	Current code | Change to
+	-----|----
+	`@ui5/webcomponents/dist/icons/add.js` | `@ui5/webcomponents-icons/dist/icons/add.js`
+	`@ui5/webcomponents/dist/icons/search.js` | `@ui5/webcomponents-icons/dist/icons/search.js`
+	etc... | etc...
+	
+	or 
+	
+	Change the import for all icons (if you were importing the whole icon collection):
+	
+	Current code | Change to
+	-----|----
+	`@ui5/webcomponents/dist/json-imports/Icons.js` | `@ui5/webcomponents-icons/dist/json-imports/Icons.js`
```

Comment:
this did not exist in rc.4 and is a new functionality

Reply:
removed it